<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/core/add_doctor.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
*{
background-color: #ecf0f1;
border-radius: 25px;

}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_patient">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
border: 5px solid #34495e;
background: #34495e;
color:white;
font-size: 30pt; 
}
QPushButton{
border: 5px solid #34495e;
background: transparent;
font-size: 20pt;
color:#34495e;
}</string>
        </property>
        <property name="text">
         <string>بیمار</string>
        </property>
        <property name="icon">
         <iconset resource="../booking/booking.qrc">
          <normaloff>:/menu/patient_menu.png</normaloff>:/menu/patient_menu.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_secretary">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
border: 5px solid #34495e;
background: #e74c3c;
color: white;
font-size: 30pt; 
}

QPushButton {
border: 5px solid #34495e;
background: transparent;
font-size: 20pt;
color: #e74c3c;
}
</string>
        </property>
        <property name="text">
         <string>منشی</string>
        </property>
        <property name="icon">
         <iconset resource="../booking/booking.qrc">
          <normaloff>:/menu/lllllllll-removebg-preview.png</normaloff>:/menu/lllllllll-removebg-preview.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_doctor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
border: 5px solid #34495e;
background: #2ecc71;
color: white;
font-size: 30pt; 
}

QPushButton {

border: 5px solid #34495e;
background: transparent;
font-size: 20pt;
color:#2ecc71;
}
</string>
        </property>
        <property name="text">
         <string>دکتر</string>
        </property>
        <property name="icon">
         <iconset resource="../booking/booking.qrc">
          <normaloff>:/menu/doc_menu.png</normaloff>:/menu/doc_menu.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../booking/booking.qrc"/>
 </resources>
 <connections/>
</ui>
